pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -92,6 +96,30 @@ type UpdateStrategy struct {
 	*RegistryPoll `json:"registryPoll,omitempty"`
 }
 
+// UnmarshalJSON implements the encoding/json.Unmarshaler interface.
+// UnmarshalJSON checks to see the provided polling interval is valid. If so, use it, if not set it to the default interval.
+func (u *UpdateStrategy) UnmarshalJSON(data []byte) error {
```

Comment:
I think I can wrap my head around why this implementation is favorable, but it wasn't immediately clear to me - any value in beefing up the description, and linking to an issue/BZ for why this workaround is necessary?